api/client/src/main/java/org/projectnessie/client/auth/oauth2/AuthorizationCodeFlow.java

```diff
@@ -64,9 +64,8 @@ class AuthorizationCodeFlow implements AutoCloseable {
   private final URI authorizationUri;
   private final Duration flowTimeout;
 
-  private final CompletableFuture<HttpExchange> requestFuture = new CompletableFuture<>();
+  private final CompletableFuture<HttpExchange> redirectUriFuture = new CompletableFuture<>();
   private final CompletableFuture<Void> closeFuture = new CompletableFuture<>();
-  private final CompletableFuture<String> authCodeFuture;
```

Comment:
We don't need this intermediary future, it is probably a leftover from previous designs.